streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/KubernetesRetriever.scala

```diff
@@ -41,11 +41,11 @@ import scala.util.{Failure, Success, Try}
 object KubernetesRetriever extends Logger {
 
   // see org.apache.flink.client.cli.ClientOptions.CLIENT_TIMEOUT}
-  val FLINK_CLIENT_TIMEOUT_SEC = 30L
+  val FLINK_CLIENT_TIMEOUT_SEC = 60L
   // see org.apache.flink.configuration.RestOptions.AWAIT_LEADER_TIMEOUT
-  val FLINK_REST_AWAIT_TIMEOUT_SEC = 10L
+  val FLINK_REST_AWAIT_TIMEOUT_SEC = 30L
   // see org.apache.flink.configuration.RestOptions.RETRY_MAX_ATTEMPTS
-  val FLINK_REST_RETRY_MAX_ATTEMPTS = 2
+  val FLINK_REST_RETRY_MAX_ATTEMPTS = 30
```

Comment:
I think it would be better to change it to 20, the default value of RestOptions.RETRY_MAX_ATTEMPTS.

Reply:
> I think it would be better to change it to 20, the default value of RestOptions.RETRY_MAX_ATTEMPTS.

I think the redundant number of times can cover unknown issues.

Reply:
I agree with you that this part affects flink task deployment and state tracking, and the state tracking part has its own additional timeout control mechanism, and increasing the retry parameter does alleviate the timeout tolerance of the submitted job.
So, LGTM